### PR TITLE
Bump up rails 6.1.7.2

### PR DESCRIPTION
**Pull request by EGiataganas**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.7.5
+          ruby-version: 2.7.7
```
Comment (benjamineskola) [inline .github/workflows/build.yml:20]: I believe this defaults to using `.ruby-version` so we could leave it out and avoid duplication.

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "2.7.5"
+ruby "2.7.7"
```
Comment (benjamineskola) [inline Gemfile:4]:
```suggestion
ruby File.read(".ruby-version").strip.split("-").last
```

Avoid duplication.

File: .github/workflows/build.yml (hunk 1 above)
Comment (EGiataganas) [reply .github/workflows/build.yml:20]: yes, that's correct! just reread their docs and says `# Not needed with a .ruby-version file`

File: Gemfile.lock [hunk 3]
```diff
@@ -347,7 +347,7 @@ DEPENDENCIES
   webpacker (~> 5.0)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 2.7.7p221
 
 BUNDLED WITH
-   2.3.12
+   2.4.6
```
Comment (benjamineskola) [inline Gemfile.lock:353]: Have we checked that this version will be compatible with the deploy hosts? I know it was an issue on AWS but I don't know if it matters with Heroku.

Comment (EGiataganas) [reply Gemfile.lock:353]: It was an issue as ansible wasn't updated

**Review by benjamineskola — APPROVED** [review]

**Pull request merged**